FarihaIS/VanRoomies
# feat(create/modify preferences): Add a User Preferences to the Profile and Adding/Modifying Functions

# Welcome to VanRoomies! 👋✈️

Fixes: #77 

## Description of the change:
- Add email and birthday to profile
- Add UserPreferences that are fetched, created, and updated in the Profile Page
- Add a button to handle creating/updating user preferences
- Information is validated before sending to the BE.

## How to manually test:
1. Start up MongoDB
2. Run `npm run dev` inside the backend folder
3. Create a new user in Postman 
4. Build and run the app
5. In the ProfileFragment you should be able to add user preferences
6. Try this with valid and invalid inputs!
7. You can also edit the preferences in the same button (which should say "Edit" now)
8. Try this with valid and invalid inputs as well!
